### PR TITLE
Add script for correcting descriptionLanguage

### PR DESCRIPTION
**Pull request by kwahlin**

### CODE REVIEW

File: whelktool/scripts/cleanups/2020/11/lxl-3413-fix-incorrect-descriptionLanguage/lxl-3413-fix-incorrect-descriptionLanguage.groovy [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+PrintWriter failedUpdating = getReportWriter("failed-updates")
+
+List ids = new File(scriptDir, 'data-1601623118108.txt').readLines().drop(1).collect{line -> line.split('\t')[1]}
+
+selectByIds(ids) { auth ->
+
+    def descriptionLang = auth.graph[0].descriptionLanguage
+
+    if (!(descriptionLang['@id'] == 'https://id.kb.se/language/swe')) {
+        descriptionLang = ['@id':'https://id.kb.se/language/swe']
```
Comment (andersju) [inline whelktool/scripts/cleanups/2020/11/lxl-3413-fix-incorrect-descriptionLanguage/lxl-3413-fix-incorrect-descriptionLanguage.groovy:11]: The record is not actually changed here - change `descriptionLang` to `auth.graph[0].descriptionLanguage`

**Review by andersju — CHANGES_REQUESTED** [review]
See above, just one little change.

File: whelktool/scripts/cleanups/2020/11/lxl-3413-fix-incorrect-descriptionLanguage/lxl-3413-fix-incorrect-descriptionLanguage.groovy (hunk 1 above)
Comment (kwahlin) [reply whelktool/scripts/cleanups/2020/11/lxl-3413-fix-incorrect-descriptionLanguage/lxl-3413-fix-incorrect-descriptionLanguage.groovy]: Fixed in 7d415dd!

**Review by andersju — APPROVED** [review]
Lookin' good now!

**Pull request merged**